akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -16,6 +16,10 @@ import akka.japi.Util
 import java.util.Comparator
 import java.util.concurrent.CompletionStage
 
+import akka.dispatch.ExecutionContexts
+import akka.stream.impl.fusing.LazyFlow
+import akka.stream.scaladsl.Flow
```

Comment:
this import of scaladsl.Flow in javadsl is wrong

Reply:
and also not used :) removed.